vite.config.ts

```diff
@@ -30,8 +30,8 @@ export default defineConfig({
     include: ['**/__tests__/*.spec.{ts,tsx}'],
     coverage: {
       lines: 90,
-      functions: 80,
-      branches: 80,
+      functions: 70,
+      branches: 70,
```

Comment:
Seems our CI checks have not been set to required for some time, allowing coverage to fall. I will remedy that and we can bump this up again.

Reply:
Oof 🫠 ... if you have a sense for which coverage would be most valuable, I'd be happy to help write tests.